lms/djangoapps/mobile_api/models.py

```diff
@@ -76,3 +81,17 @@ def save(self, *args, **kwargs):
         """ parses version into major, minor and patch versions before saving """
         self.major_version, self.minor_version, self.patch_version = utils.parsed_version(self.version)
         super(AppVersionConfig, self).save(*args, **kwargs)
+
+
+class IgnoreMobileAvailableFlagConfig(ConfigurationModel):  # pylint: disable=W5101
+    """
+    Configuration for the mobile_available flag. Default is false.
+
+    Setting mobile_available_override to true will always grant access in
```

Comment:
Nit: Need to update this docstring since `mobile_available_override` no longer exists.